website/src/react-native/css/react-native.css

```diff
@@ -1013,10 +1013,24 @@ h2 {
   text-rendering: optimizelegibility; }
 
 .docs-prevnext {
-  min-width: 320px;
   max-width: 640px;
   margin: 40px auto;
-  padding-bottom: 20px; }
+  padding-bottom: 20px;
+}
+
+.docs-prevnext:before,
+.docs-prevnext:after {
```

Comment:
From my small experimenting in the chrome console, it seems like your media query addition is the only part of the PR needed to fix the previous/next button of the docs. Why do we need this display: table stuff?